src/components/Patient/PatientLaboTable/PatientLaboTable.tsx

```diff
@@ -27,17 +28,30 @@ const PatientLaboTable = ({ patient }: PatientLaboTableProps) => {
 
   const [page, setPage] = useState(1)
 
+  const labResultValue = (res: IObservation): string => {
+    if (res.valueQuantity) return `${res.valueQuantity.value} ${res.valueQuantity.unit}`
+    if (res.valueBoolean) return `${res.valueBoolean}`
+    if (res.valueString) return res.valueString
+    if (res.valueTime) return res.valueTime
+    if (res.valueDateTime) return res.valueDateTime
+    if (res.valueInteger) return res.valueInteger.toString()
+    if (res.valueRange?.low || res.valueRange?.high)
+      return `${res.valueRange.low?.value} ${res.valueRange.low?.unit} - ${res.valueRange.high?.value} ${res.valueRange.high?.unit}`
+    if (res.valuePeriod) return `${res.valuePeriod.start} - ${res.valuePeriod.end}`
+    if (res.valueRatio?.numerator || res.valueRatio?.denominator)
+      return `${res.valueRatio.numerator?.value} ${res.valueRatio.numerator?.unit} / ${res.valueRatio.denominator?.value} ${res.valueRatio.denominator?.unit}`
+    if (res.valueCodeableConcept?.text) return res.valueCodeableConcept.text
```

Comment:
@nriss maybe we can integrate this to Cohort360 IG. We can add a MustSupport on the text attribute if valueCodeableConcept is filled, to make sure something will appear on the app.

Reply:
Yep! 
Also discussed there https://github.com/arkhn/Cohort360/issues/95 about which attribute to show, I suggested `display > text > code` depending on which attribute is filled. The order `text > display > codeù seems also great